pysb/export/sbml.py

```diff
@@ -170,17 +169,20 @@ def convert(self, level=(3, 2)):
 
         # Initial values/assignments
         initial_concs = [0.0] * len(self.model.species)
-        for cp, param in self.model.initial_conditions:
-            sp_idx = self.model.get_species_index(cp)
-            if isinstance(param, pysb.Expression):
+        fixed_species_idx = set()
+        for ic in self.model.initials:
+            sp_idx = self.model.get_species_index(ic.pattern)
+            if isinstance(ic.value, pysb.Expression):
                 ia = smodel.createInitialAssignment()
                 _check(ia)
                 _check(ia.setSymbol('__s{}'.format(sp_idx)))
-                init_mathml = self._sympy_to_sbmlast(sympify(param.name))
+                init_mathml = self._sympy_to_sbmlast(ic.value)
```

Comment:
sympify wasn't even needed here, the param object *is* a `sympy.Symbol` already.